ocs_ci/deployment/deployment.py

```diff
@@ -1626,8 +1649,24 @@ def deploy_with_external_mode(self):
         # enable secure connection mode for in-transit encryption
         if config.ENV_DATA.get("in_transit_encryption"):
             external_cluster.enable_secure_connection_mode()
+        if config.DEPLOYMENT.get("multi_storagecluster"):
+            logger.info("not setting toolbox in multi-storagecluster")
+            health_check_namespace = constants.OPENSHIFT_STORAGE_EXTENDED_NAMESPACE
+        else:
+            setup_ceph_toolbox()
+            health_check_namespace = constants.OPENSHIFT_STORAGE_NAMESPACE
```

Comment:
```suggestion
            health_check_namespace = None
```
Let the ceph_healt_check function dertermin itslef the namespace form config in this case

Reply:
Dropped the condition for health_check_namespace and sending ceph_health_check with no parameters.